Earthfile

```
@@ -0,0 +1,41 @@
+VERSION 0.6
+
+equivs:
+  FROM ubuntu:jammy
+  RUN apt-get update
+  RUN apt-get install -y equivs
+
+rpmbuild:
+  FROM almalinux:8
+  RUN dnf install -y rpmdevtools
+
+RPM:
+  COMMAND
+  ARG package
+  FROM +rpmbuild
+  RUN echo "!!! WARNING THIS DOESN'T WORK YET !!!"
+  COPY $package /tmp/$package
+  WORKDIR /tmp/$package
+  RUN rpmbuild -ba ${package}.spec
```

Among the things I like least about `rpmbuild` is that it really doesn't care about your current directory at all, and wants to do all of it's operations in a pre-configured staging directory which defaults to `~/rpmbuild/`. Regardless of where the spec file resides, it expects inputs to be under that directory and puts outputs there too (by default).

Many wrappers around `rpmbuild` change this behavior, and so can we.
```suggestion
  RUN rpmbuild -ba ${package}.spec --define "_topdir /tmp/$package/rpm" --define "_sourcedir /tmp/$package"
```

These flags will direct `rpmbuild` to use `/tmp/$package/rpm` instead of `~/rpmbuild/`, and also look in `/tmp/$package` for input files rather than `/tmp/$package/rpm/SOURCES`.

After invoking the build, it might be a good idea to invoke `rpmlint` on the outputs. There are several warnings and errors from that tool that we should address regarding these spec files.

---

> After invoking the build, it might be a good idea to invoke `rpmlint` on the outputs. There are several warnings and errors from that tool that we should address regarding these spec files.

Good idea. Added both rpmlint and lintian to their respective build pipelines in https://github.com/ros-infrastructure/infra-variants/pull/1/commits/585bf1d8042bb7d0af99827dd1e240e32a8896de

The identified warnings and errors have been addressed in subsequent commits.